scrapi/harvesters/dataone.py

```diff
@@ -20,18 +20,135 @@
+            'authorSurName': ("str[@name='authorSurName']/node()"),
+            'authoritativeMN': ("str[@name='authoritativeMN']/node()"),
+            'checksum': ("str[@name='checksum']/node()"),
+            'checksumAlgorithm': ("str[@name='checksumAlgorithm']/node()"),
+            'dataUrl': ("str[@name='dataUrl']/node()"),
+            'datasource': ("str[@name='datasource']/node()"),
+            'documents': "arr[@name='documents']/str/node()",
+            'dateModified': ("date[@name='dateModified']/node()"),
+            'datePublished': ("date[@name='datePublished']/node()"),
+            'dateUploaded': ("date[@name='dateUploaded']/node()"),
+            'pubDate': ("date[@name='pubDate']/node()"),
+            'updateDate': ("date[@name='updateDate']/node()"),
+            'fileID': ("str[@name='fileID']/node()"),
+            'formatId': ("str[@name='formatId']/node()"),
+            'formatType': ("str[@name='formatType']/node()"),
+            'identifier': ("str[@name='identifier']/node()"),
+            'investigator': "arr[@name='investigator']/str/node()",
+            'origin': "arr[@name='origin']/str/node()",
+            'isPublic': ("bool[@name='isPublic']/node()"),
+            'readPermission': "arr[@name='readPermission']/str/node()",
+            'replicaMN': "arr[@name='replicaMN']/str/node()",
+            'replicaVerifiedDate': "arr[@name='replicaVerifiedDate']/date/node()",
+            'replicationAllowed': ("bool[@name='replicationAllowed']/node()"),
+            'numberReplicas': ("int[@name='numberReplicas']/node()"),
+            'preferredReplicationMN': "arr[@name='preferredReplicationMN']/str/node()",
+            'resourceMap': "arr[@name='resourceMap']/str/node()",
+            'rightsHolder': ("str[@name='rightsHolder']/node()"),
+            'scientificName': "arr[@name='scientificName']/str/node()",
+            'site': "arr[@name='site']/str/node()",
+            'size': ("long[@name='size']/node()"),
+            'sku': ("str[@name='sku']/node()"),
+            'isDocumentedBy': "arr[@name='isDocumentedBy']/str/node()",
+        },
+        'contributors': ("str[@name='author']/node()", "str[@name='submitter']/node()", "arr[@name='origin']/str/node()", process_contributors),
+        'id': {
+            'serviceID': "str[@name='id']/node()",
+            'doi': ("str[@name='id']/node()", "arr[@name='isDocumentedBy']/str/node()", process_doi),
+            'url': ("str[@name='id']/node()", "//str[@name='dataUrl']/node()", lambda x, y: y if 'http' in y else x if 'http' in x else '')
+        },
+        'tags': ("//arr[@name='keywords']/str/node()", lambda x: x if isinstance(x, list) else [x]),
+        'dateUpdated': ("str[@name='dateModified']/node()", lambda x: unicode(parse(x).isoformat())),
+        'title': "str[@name='title']/node()",
+        'description': "str[@name='abstract']/node()"
+    }
 
     DEFAULT_ENCODING = 'UTF-8'
```

Comment:
Or rather, we as a group should re-take a look at the copy to unicode function! This is OK as is - my bad!